### PR TITLE
Feature calculator design

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
add calculator.js file
this file include the basic structure of the calculator component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (uwadonat) [inline src/App.js]: - [ ] App component should be class-based.

File: README.md [hunk 2]
```diff
@@ -1,65 +1,72 @@
...
+# React - Math Magicians (Calculator)
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Set-up GitHub Actions
+# Math Magicians
 
-This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+> This application use React framework to show a calculator.
```
Comment (uwadonat) [inline README.md]: - [ ] Your live demo link is pointing to an other project. Please double check
Screenshot from 2022-05-25 11-02-52

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <div className="cell calc-plus-minus">+-</div>
+          <div className="cell calc-mode">%</div>
+          <div className="cell calc-division">/</div>
+          <div className="cell calc-seven">7</div>
+          <div className="cell calc-eight">8</div>
+          <div className="cell calc-nine">9</div>
+          <div className="cell calc-multiply">*</div>
+          <div className="cell calc-four">4</div>
+          <div className="cell calc-five">5</div>
+          <div className="cell calc-six">6</div>
```
Comment (uwadonat) [inline src/components/Calculator.js]: - [x] Kindly make sure your design match the given one.
- - Your design
yours
- Given design
Screenshot from 2022-05-25 11-01-50

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[maenkhraisha](https://github.com/maenkhraisha),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- No linter errors  ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 3 above)
Comment (maen-al-khraisha) [reply src/components/Calculator.js]: thank you for your feedback,
but what the problem with the mod symbol?

Comment (uwadonat) [reply src/components/Calculator.js]: Hi! for mod symbol it's ok. sorry for that.

**Review by Meltrust — APPROVED** [review]
Hi @maenkhraisha,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To highlight

- Thank you for correcting your readme, the symbols on the calculator, and for refactoring your App component to a class✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**